build/helper/codegen_helper.py

```diff
@@ -67,7 +71,7 @@ def get_params_snippet(function, parameter_usage_options):
     return ', '.join(snippets)
 
 
-def _get_output_param_return_snippet(output_parameter, parameters, config):
+def _get_output_param_return_snippet(output_parameter, parameters, config, use_numpy_array=False):
```

Comment:
`use_numpy_array` isn't used in this function

Reply:
Fixed